modules/drivers/mongo/src/metabase/driver/mongo/parameters.clj

```diff
@@ -108,6 +116,22 @@
       (params/FieldFilter? v)
       (let [no-value? (= (:value v) params/no-value)]
         (cond
+          (ops/operator? (get-in v [:value :type]))
+          (let [param (:value v)
+                compiled-clause (-> (assoc param
+                                           :target
+                                           [:template-tag
+                                            [:field (field->name (:field v) false)
+                                             {:base-type (get-in v [:field :base_type])}]])
+                                    ops/to-clause
+                                    ;; desugar only impacts :does-not-contain -> [:not [:contains ... but it prevents
+                                    ;; an optimization of [:= 'field 1 2 3] -> [:in 'field [1 2 3]] since that
+                                    ;; desugars to [:or [:= 'field 1] ...].
+                                    (mbql.u/desugar-filter-clause)
```

Comment:
You don't need to wrap these... you can just do `mbql.u/desugar-filter-clause`

Reply:
heh. every time i write these i vacillate back and forth between wrapped and unwrapped. i'll stick with unwrapped from now on